configs/modules/t31n_sc2332_hi3881

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
please remove the space at the top =D

Reply:
done

Reply:
this PR need https://github.com/gtxaspec/u-boot-ingenic/pull/43 to be accepted first